travel-agency/public/js/ajax-city-handler.js

```diff
@@ -0,0 +1,98 @@
+$.ajaxSetup({
+    headers: {
+        'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+    }
+});
+$('form').submit(function(event){
+    event.preventDefault();
+})
+function createCity(){
+    var currentPage = new URLSearchParams(window.location.search).get('page') || 1;
+    var jsonData = formToJson($('#add-city-form'),currentPage);
```

Comment:
- Use `var` when you want function or global scope variables and are not concerned about block scoping. However, it's generally recommended to avoid using var in modern JavaScript code due to its scoping quirks.
- Use `let` when you need block-scoped variables that can be reassigned. It's a safer alternative to var for many scenarios.
- Use `const` when you need block-scoped variables that won't be reassigned. This is a good choice for values that should remain constant throughout their scope.

Modern JavaScript development encourages the use of `let` and `const` due to their more predictable scoping behavior and improved clarity in code.

Let's not use `var` because of what's explained above, and instead replace it with either `let` or `const`. In this case, you should use `const` because `jsonData` isn't something that should be possible to override/change (if it should be possible to do that, you should use `let` instead).